ddtrace/settings/config.py

```diff
@@ -729,6 +729,23 @@ def __init__(self, callback):
 
         return _GlobalConfigPubSub
 
+    def _tracerflarePubSub(self):
```

Comment:
This was mainly because I thought it would be more intuitive to have dedicated pubsubs for different tasks. 

I wanted a separate pubsub for the `AGENT_TASK` product because there would be enough distinct logic for that product that would warrant its own callback IMO. I was a bit on the fence for `AGENT_CONFIG`, especially given the most recent comment on the RFC (tl;dr - we'll probably change the trigger to be `APM_TRACING` instead of `AGENT_CONFIG` in the long term, but for now it'll be `AGENT_CONFIG`). Given that I would already be creating the new pubsub for `AGENT_TASK`, I thought to just bundle `AGENT_CONFIG` with it as a tracer flare pubsub.

I'm definitely open to consolidating them into the existing `_remoteconfigPubSub` (especially given that we're defaulting to enabling the flares if RC is enabled).

Reply:
Chatted offline. Will consolidate them for now, and if it looks like the logic for handling the `AGENT_TASK` product becomes too large, we can separate them out for readability (not doing this now since the pubsubs have a lot of duplicate boilerplate code)